spec/origin_info_spec.rb

```diff
@@ -249,6 +249,11 @@
             @mods_rec.from_str("<mods #{@ns_decl}><originInfo><dateCreated>other date</dateCreated><dateCreated keyDate='yes'>key date</dateCreated></originInfo></mods>")
             expect(@mods_rec.origin_info.as_object.first.key_dates.first.text).to eq 'key date'
           end
+          it 'should extract a date range when the keyDate attribute is on the start of the range' do
```

Comment:
Should it?  Can we use present tense? http://www.betterspecs.org/#should

Reply:
The rest of the tests use `should` right now, so this seems consistent with the practices of this codebase.  Agreed that not using `should` is better, but this can be handled in a large spec refactor in the future.